### PR TITLE
Doctests for feature-importance example notebook

### PR DESCRIPTION
**Pull request by ivarpruijn**
Reformats the feature-importance example notebook, and adds basic doctests. It does not yet full doctest most functionality, as [the PR for this model is still a WIP](https://github.com/objectiv/objectiv-analytics/pull/662/).

To do:

- [x] Add example output.

### CODE REVIEW

**Comment by ivarpruijn** [comment]
@hendrik-obj I did not update the Jupyter notebook in https://github.com/objectiv/objectiv-analytics/pull/662/, as I know you're actively working on that PR. I just updated the .rst file here.

File: bach/docs/source/example-notebooks/open-taxonomy.rst [hunk 1]
```diff
@@ -53,6 +53,7 @@ in the global contexts and how to access this data for analyses.
 
 	start_date = '2022-06-01'
 	end_date = '2022-06-30'
+	pd.set_option('display.max_colwidth', 93)
```
Comment (ivarpruijn) [inline bach/docs/source/example-notebooks/open-taxonomy.rst:56]: Small change to make sure that doctests don't fail on this if they for some reason first run the open-taxonomy notebook.

**Comment by ivarpruijn** [comment]
Output shown here: https://github.com/objectiv/objectiv.io/pull/562

**Review by vard-obj — APPROVED** [review]
lgtm

**Comment by ivarpruijn** [comment]
@hendrik-obj As discussed with @vard-obj this morning, I'll leave it up to you to decide how much of these changes should be ported to the `feature-importance.ipynb` notebook in your PR. Will merge this PR for now.

**Pull request merged**